L1Trigger/TrackFindingTMTT/interface/Array2D.h

```diff
@@ -2,6 +2,7 @@
 #define L1Trigger_TrackFindingTMTT_Array2D_h
 
 #include <vector>
+#include <stdexcept>
```

Comment:
is this to pass the header consistency checks ?

Reply:
I now deleted stdexcept, and changed Array2D to throw a cms::Exception instead.